components/global/mixer-adapter/internal/helper.go

```diff
@@ -0,0 +1,5 @@
+package internal
+
+const GrpcAdapterCredential string = "GRPC_ADAPTER_CREDENTIAL"
```

Comment:
Better to move these constants into `wso2spadapter.go` file since it is only used there. (Normally in GoLang we do not keep separate files for constants.)